guides/_subsections/archive-a-recording.md

```diff
@@ -41,14 +41,14 @@ any state (`RUNNING`, `STOPPED`, etc.), and may even be a snapshot.
       summary="Navigate to the <i>Archived Recordings</i> tab"
       image-name="2.3.0/archive-a-recording-2.png"
       caption="
-        Once the recording has been archived, a new entry will appear in the
-        target JVM's <i>Archived Recordings</i> table. All recordings that were
+        Once the <code>Recording</code> has been <code>Archived</code>, a new entry will appear in the
+        target <b>JVM's</b> <i>Archived Recordings</i> table. All <code>Recordings</code> that were
         saved from the current target will be listed here in their own table.
-        Switching to a different target from the dropdown will list only the 
-        recordings archived from that source target. The name of the archived 
-        recording reflects the address of the target application, the original 
-        name of the active recording that it was retrieved from, and includes a 
-        timestamp indicating when the archived recording was created.
+        Switching to a different target from the dropdown will list only the
+        <code>Recordings</code> <code>Archived</code> from that source target. The name of the <code>Archived
```

Comment:
same here as above- could just have "recording" in the code block because "archived" is more of a verb